loopdb/loopout.go

```diff
@@ -24,6 +24,14 @@ type LoopOutContract struct {
 	// DestAddr is the destination address of the loop out swap.
 	DestAddr btcutil.Address
 
+	// DestAmount specifies the exact amount to send to DestAddr. Optional.
+	// Used together with ChangeAddr.
+	DestAmount btcutil.Amount
+
+	// Where to send change in case DestAmount is specified. Optional.
+	// Used together with DestAmount.
+	ChangeAddr btcutil.Address
```

Comment:
Should `deserializeLoopOutContract` and `serializeLoopOutContract` be updated?

They use some custom binary format. I guess the change of the format should be backward-compatible, so that strings produced by old `serializeLoopOutContract` are successfully decoded by new `deserializeLoopOutContract`. I'll add two new fields in the end and in `deserializeLoopOutContract` if I get EOF trying to read the first field, I'll consider that ok, because it is an old format.

Reply:
>  if I get EOF trying to read the first field, I'll consider that ok, because it is an old format.

I see migration files. I think, I should add a migration.

Reply:
Done